src/tlo/methods/healthsystem.py

```diff
@@ -584,6 +598,12 @@ def pre_initialise_population(self):
 
         # Determine mode_appt_constraints
         self.mode_appt_constraints = self.get_mode_appt_constraints()
+        
+        # Determine year mode switch
+        self.year_mode_switch = self.get_year_mode_switch()
+
+        # Determine year mode switch
+        self.scale_to_effective_capabilities = self.get_scale_to_effective_capabilities()
```

Comment:
```suggestion
```
If changing the `get_*` methods to properties as suggested below these definitions can be removed.

Reply:
Depending on whether you are viewing this in conversation view or file view on GitHub 'below' might actually be above 😅